### PR TITLE
Coins/markets function

### LINKED ISSUE #21: Add a function for /coins/markets coingecko endpoint
For more details look at #4

### PR DESCRIPTION
**Pull request by WHYTEWYLL**
Fixes #21

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (Niraj-Kamdar) [inline .vscode/extensions.json:1]: Can you add this in .gitignore

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json (hunk 1 above)
Comment (WHYTEWYLL) [reply .vscode/extensions.json]: Doneª

File: packages/coingecko/recipes/endpoints/coinsMarket/coinsMarket.graphql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+query coinsMarkets (
```
Comment (Niraj-Kamdar) [inline packages/coingecko/recipes/endpoints/coinsMarket/coinsMarket.graphql:1]: rename this to `schema.graphql`

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]
It has to be named `schema.graphql` and `e2e.json`

File: packages/coingecko/recipes/endpoints/coinsMarket/coinsMarket.graphql (hunk 2 above)
Comment (WHYTEWYLL) [reply packages/coingecko/recipes/endpoints/coinsMarket/coinsMarket.graphql]: Done, and pushed!

File: packages/coingecko/src/query/schema.graphql [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+    order:String
+    per_page: Int
+    page: Int
+    sparkline: Boolean
+    price_change_percentage: [String!]
+    ):[CoinsMarkets!]!
 
-  simplePrice(
+
+    simplePrice(
```
Comment (Niraj-Kamdar) [inline packages/coingecko/src/query/schema.graphql:19]: format this. feel free to look into auto formatters for graphql if you are interested

**Pull request merged**